internal/store/projects.go

```diff
@@ -194,7 +194,7 @@ func (s *Store) AddTeamsToProject(projectID string, teams []*atlas.ProjectTeam)
 // DeleteTeamFromProject encapsulates the logic to manage different cloud providers.
 func (s *Store) DeleteTeamFromProject(projectID, teamID string) error {
 	switch s.service {
-	case config.CloudService:
+	case config.CloudService, config.CloudGovService:
```

Comment:
[q] what's the plan here, do you plan to test command by command to enable api access? it may be easier to enable for all and then start restricting but want to hear your thoughts

Reply:
I was thinking about creating another ticket where I test and enable all the commands for cloud-gov. Yes, I would enable  for all in my local machine (except the ones that we are sure are not there -> datalakes etc) and see if they work but add in my PR only the ones that I am sure are working